src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx

```diff
@@ -40,6 +40,7 @@ Potential permissions-related factors that can affect your access:
 If you're logged in but can't find an expected account or UI page, it may be because you're in the wrong account in your organization. Some tips to troubleshoot this:
 
 * If your organization has multiple accounts and you have access to those accounts, you can switch to another account by going to the account switcher at the top left of most UI pages.
+* If you have multiple user records (for example, access to more than one New Relic organization), you can [switch between your user records](/docs/accounts/accounts-billing/account-setup/multiple-logins-found). 
```

Comment:
@zuluecho9 Why say "user records" here? Why not "user accounts"?

Reply:
'account' and 'user record' is an entirely different concept. There can be multiple 'accounts' in a New Relic organization. And a user can have access to multiple accounts with the same login.